packages/fluentui/react-northstar/src/components/Form/FormButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import Button, { ButtonProps } from '../Button/Button';
+import FormFieldCustom, { FormFieldCustomProps, FormFieldCustomStylesProps } from './FormFieldCustom';
+
+interface FormButtonOwnProps extends Omit<ButtonProps, 'styles' | 'accessibility'> {}
+
+export interface FormButtonProps extends FormFieldCustomProps, FormButtonOwnProps {}
+export type FormButtonStylesProps = never;
+
+export const FormButtonClassName = 'ui-form-button';
+
+const FormButton = compose<
+  'div',
```

Comment:
```suggestion
  'button',
```

Reply:
Don't think so, we still composing with `FormCustomField` so it will be a div and the button is `children`

Reply:
Hm, you're right 👍